package SellComponentConfig;

import "RewardConfig.proto";

message GoodInfo
{
	required uint32 price	= 1;
	required uint32 times	= 2;
	repeated RewardConfig.Reward goods	= 3; //商品
}

message SellComponent
{
	repeated GoodInfo goodsall	= 1;
}